### PR TITLE
feat: Add provided gender to config file & update rules

### PR DESCRIPTION
**Pull request by ivadym**
### This PR:

- Adds provided gender to config file
- Updates following rules including the provided gender as an input:
  - `vcf2cytosure_convert` (TO-PANEL)
  - `vcf2cytosure_convert_tumor_only`, `vcf2cytosure_convert_tumor_normal` (WGS)
  - `cnvkit_single`, `cnvkit_paired` (PANEL)
  - `ascat_tumor_normal` (TN-WGS)

### Review and tests: 
- [ ] Tests pass
- [ ] Code review
- [ ] New code is executed and covered by tests, and test approve

### CODE REVIEW

**Comment by ivadym** [comment]
### Tests:

- Dry run:

Screen Shot 2022-06-13 at 15 01 06


- `config.json`

Screen Shot 2022-06-13 at 14 59 35

**Review by ashwini06 — CHANGES_REQUESTED** [review]
Except for your changes to the `run_validate.sh`, the rest of the changes related to the gender option looks good  💯 
Unless no specific reason for changing this test script, I would suggest reverting it back to the original version

File: run_validate.sh [hunk 1]
```diff
@@ -10,9 +10,8 @@ _nocol='\033[0m';
 
 function usage() {
   echo $"
-USAGE: [ -a <T|TN> -c _condaenv -m <run|config|all> -t <panel|WGS> -r ]
+USAGE: [ -c _condaenv -m <run|config|all> -t <panel|WGS> -r ]
```
[the remark this answers is not in the document]
Comment (ivadym) [reply run_validate.sh:13]: The `--analysis-type` flag has been removed:

> CHANGELOG.rst
>
> Removed cli --analysis_type for balsamic report deliver command and balsamic run analysis https://github.com/Clinical-Genomics/BALSAMIC/pull/942

[the remark this answers is not in the document]
Comment (ashwini06) [reply run_validate.sh:13]: Indeed, I forgot about that change 😜

**Review by ashwini06 — APPROVED** [review]

**Comment by ivadym** [comment]
Screen Shot 2022-06-17 at 09 57 30

**Pull request merged**